state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -752,6 +810,26 @@ pub mod pallet {
 					Self::recycle_channel(&mut used_weight, address);
 				}
 			}
+
+			// A report gets cleaned up after approx 1 hour and needs to be re-reported by the
```

Comment:
What if the broker re-reports? Would it then overwrite the `BoostRejected` record? (This would be problematic.)